### PR TITLE
Skip parsing station details if turned off

### PR DESCRIPTION
**Pull request by Mygod**

### CODE REVIEW

**Comment by jfberry** [comment]
Good addition

**Comment by Fabio1988** [comment]
Thanks, good catch :) So I added it to GMO, but not to the details method :)

File: main.go [hunk 1]
```diff
@@ -435,8 +435,10 @@ func decode(ctx context.Context, method int, protoData *ProtoData) {
 		}
 		break
 	case pogo.Method_METHOD_GET_STATION_DETAILS:
-		// Request is essential to decode this
```
Comment (Ppeepost4489) [inline main.go:438]: main.go

File: main.go
Comment (Ppeepost4489) [inline main.go:1]: main.go

**Review by Ppeepost4489 — COMMENTED** [review]
main.go

**Comment by Ppeepost4489** [comment]
> 

main.go

**Pull request merged**